backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -825,7 +825,11 @@ public async System.Threading.Tasks.Task<Project> SetStatusAsync(Project project
             project.CopyRowVersionTo(originalProject);
             this.Context.CommitTransaction();
 
-            await SendNotificationsAsync(originalProject, fromStatusId, responses, noteChanged);
+
+            await SendNotificationsAsync(originalProject, fromStatusId, responses, noteChanged, project.SendNotifications);
+            // Set back to default true after toggle has been passed
```

Comment:
This is what I am not really a fan of with the toggle being on the project

Reply:
Instead of this, add an options property for the `SetStatusAsync(...)` function that allow for controlling whether notifications are sent.  Then wrap this `SendNotificationsAsync(...)` with an `if (options.SendNotifications) {...}`